README.md

```diff
@@ -118,6 +124,15 @@ Parameters:
 - `role`: Name of the role, e.g: `caseworker-divorce`.
 - `classification`: Optional. One of `PUBLIC`, `PRIVATE` or `RESTRICTED`. Defaults to `PUBLIC`.
 
+#### Note:
+If you get `{"timestamp":"2019-07-16T09:11:57.403+0000","status":403,"error":"Forbidden","message":"Access Denied","path":"/api/user-role"}` error you may need to install the `jq` library like so:
+```bash
+brew update;
+brew install jq;
+```
+
+If you are using Windows you should buy a Mac at this point :)
```

Comment:
There are windows users working with a win release of jq.
jq is already listed in the required tools above. And, installation link is already provided.
So, this part may not be so necessary.

Reply:
yes noted, will remove this line